### DISCUSSION TITLE
AC and H/H field do not show input text

### DISCUSSION THREAD

**Discussion by heesuallykim** [broadinstitute/seqr#3050] [Q&A]
**Describe the bug**
A clear and concise description of what the bug is, and what you instead expect to be happening

I noticed that, when I get to the variant search page, some of the "customize search" fields (including AC, H/H, etc) don't show what text is being input. I was trying to limit AC to 10 but when I typed "10", the field looked blank and I didn't know if anything was being registered.

**Link to page(s) where bug is occurring**
https://seqr.broadinstitute.org/...

https://seqr.broadinstitute.org/variant_search/results/237812f66356297b10e870349b2fdc89?page=1&sort=xpos

**Scope of the bug**
Are you experiencing this issue in all projects? Within specific families within one project?

I haven't tried this in other projects, but have done this for NeuroDev. The variant search page, though, seems like it applies to all projects.

**Screenshots**

<img width="1143" alt="image" src="https://user-images.githubusercontent.com/92403528/200221258-15159f95-2d8e-4272-9ed6-4e55f1b226f5.png">

**Comment by hanars** [comment]
Its just an issue with your screen width, if you make the screen wider you will see the value

**Comment by heesuallykim** [comment]
I did try making the screen wider — tried full screen, etc — but still didn’t show the values.

**Comment by heesuallykim** [comment]
ah, as a note, I am using the smaller Macbook Pro (the Broad-given one) as my work laptop.

**Reply by hanars** [reply]
yes, thats too small a screen. If you want, you can submit a feature request for this to show up on smaller screens, but it not considered a bug

**Comment by heesuallykim** [comment]
Oh, ok. I guess device compatibility is a thing that might need more attention later on as others in the field start using seqr, but perhaps not a priority now. Thanks for letting me know.

**Reply by hanars** [reply]
Yeah, I am happy to make changes to support more devices as people find device-specific issues, I just need to triage them as change requests and not as bugs that need urgent fixing
